src/Xpring/xpring-error.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Types of errors that originate from Xpring.
+ */
+export enum XpringErrorType {
+  MismatchedNetworks,
+}
+
+/**
+ * Represents errors thrown by PayID components of the Xpring SDK.
+ */
+export default class XpringError extends Error {
+  /** 
+   * Input entities given to a Xpring component were attached to different networks. 
+   *
+   * For instance, this error may be thrown if a XpringClient was constructed with 
+   * a PayIDClient attached to Testnet and a XRPClient attached to Mainnet.
```

Comment:
I think "`XRP`" should always be preceded by "an" instead of "a" so we don't sound dumb in the mind's ear of other devs.

Reply:
Though if we pronounce this "zerp" I guess we're already good

Reply:
Done.